app/uk/gov/hmrc/gform/controllers/AuthenticatedRequestActions.scala

```diff
@@ -165,6 +163,31 @@ class AuthenticatedRequestActions(
       } yield result
     }
 
+  def asyncWithoutObligations(
```

Comment:
`asyncWithoutObligations` is almost the same like `async`. There must be way how to generalise these two into common function @kylemchownhmrc .